utilities_common/netstat.py

```diff
@@ -49,8 +49,13 @@ def ns_util(newstr, oldstr, delta, port_rate=PORT_RATE):
         return STATUS_NA
     else:
         rate = int(ns_diff(newstr, oldstr).replace(',',''))/delta
-        util = rate/(port_rate*1024*1024*1024/8.0)*100
-        return "{:.2f}%".format(util)
+        util = rate/(port_rate*1000*1000*1000/8.0)*100
+        if util < 100:
```

Comment:
I dont think this is right way

Reply:
Any suggestion, If we don't round-off, value sometimes looks like 100.05%. This is because of time delta is not accurate.

Reply:
If you 1024, if should would not be more than 100%

Even with 1024, if it is more than 100%, then the overhead crosses the link speed itself.